packages/compass-components/src/components/breadcrumb.tsx

```diff
@@ -77,9 +82,8 @@ export const Breadcrumbs = ({
           );
         }
         return (
-          <>
+          <span key={item.name} className={breadcrumbEntryStyles}>
```

Comment:
You can put keys on fragments, you just need to explicitly use `Fragment` component from React

Reply:
ok. I'll update. that's better :)